lualib/resty/events/protocol.lua

```diff
@@ -154,6 +167,11 @@ function _Client.connect(self, addr)
         end
     end -- subsystem == "http"
 
+    local _, err = send_frame(self, tostring(worker_id() or -1))
```

Comment:
In the current implementation, the broker is also the client of itself. Should we consider this situation?

For example, we can check if `self.broker_id` is equal to `worker_id()`.

Reply:
I don’t think that needs any special treatment.

Reply:
And it keeps code cleaner when broker worker also connects to broker.

Reply:
Make sense.

Reply:
The `or -1` is what this library is to give `privileged agent` a number that is normally a `nil`.